src/com/hillel/Main.java

```diff
@@ -3,6 +3,6 @@
 public class Main {
 
     public static void main(String[] args) {
-        System.out.println("Hello Java");
+        System.out.println("Hello Java Java Java");
```

Comment:
Please, make your changes in your separate package (name it with your name)

Reply:
Done